### PR TITLE
[MBL-16510][Teacher] Added Teacher Pandata

### PR DESCRIPTION
**Pull request by hermannakos**
Test plan: Using a proxy check if all the events are reported correctly. Use the `adb shell cmd jobscheduler run -f com.instructure.teacher 188372` command to trigger the service.

refs: MBL-16510
affects: Teacher
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: apps/teacher/src/main/java/com/instructure/teacher/fragments/InboxFragment.kt
[the remark this answers is not in the document]
Comment (hermannakos) [reply apps/teacher/src/main/java/com/instructure/teacher/fragments/InboxFragment.kt:1]: Probably found its way back when I was resolving conflicts

File: apps/teacher/src/main/java/com/instructure/teacher/activities/SpeedGraderActivity.kt [hunk 1]
```diff
@@ -74,14 +76,18 @@ import org.greenrobot.eventbus.Subscribe
 import org.greenrobot.eventbus.ThreadMode
 import java.util.*
 
+///courses/20820/gradebook/speed_grader?assignment_id=64155&student_id=96471
```
[the remark this answers is not in the document]
Comment (hermannakos) [reply apps/teacher/src/main/java/com/instructure/teacher/activities/SpeedGraderActivity.kt:79]: We don't, it was just the template for the page view event and forgot to delete it.

**Comment by tamaskozmer** [comment]
QA + 1

**Review by tamaskozmer — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by kristofnemere — APPROVED** [review]
QA+1

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request merged**